### ISSUE TITLE
Release v0.2 version of EVCouplings

### ISSUE THREAD

**Issue by rmaguire31** [debbiemarkslab/EVcouplings#310]
Hi

Appreciate the great work!

Recently, we noticed the v0.2 tag from your Github repository has been removed, which we were using in a venv for some of the latest fixes. We would prefer not to specify the develop branch as a dependency due to the possibility of breaking changes being introduced in the future.

I think the solution to remedy this would be to release the version you currently have defined as 0.2 in your setup.py to PyPI, which is currently still just version 0.1.1. Would you be able to release a version 0.2.0 to make using this tool more reproducible?

Thank you

**Comment by thomashopf** [comment]
Yes this is the plan as soon as I have enough evidence that current develop branch version is stable enough to be released, should be very soon!

**Comment by thomashopf** [comment]
We are finally there - just released v0.2.1 to PyPI.

 Thank you for your patience, and please feel free to reopen this issue if you encounter any problems with the latest release!
